README.md

```diff
@@ -36,10 +36,6 @@ The main focus for this initial release is to improve the developer experience w
     * [config](#config)
     * [open](#open)
   * [Developing the Stripe CLI](#developing-the-stripe-cli)
```

Comment:
Should this point straight to the wiki instead of an anchor in the readme?

Reply:
Addressed in https://github.com/stripe/stripe-cli/pull/262